src/App.tsx

```diff
@@ -1,9 +1,31 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const movies = moviesFromServer.filter(movie => {
+    const {
+      title,
+      description,
+    } = movie;
+    const lowerQuery = query.toLowerCase().trim();
+
+    if (
+      title.toLowerCase().includes(lowerQuery)
+        || description.toLowerCase().includes(lowerQuery)
+    ) {
+      return true;
+    }
+
+    return false;
+  });
```

Comment:
You can simplify that :)
```suggestion
  return title.toLowerCase().includes(lowerQuery) 
    || description.toLowerCase().includes(lowerQuery);
  });
```